### PR TITLE
Add Statistical Mechanics

### PR DESCRIPTION
**Pull request by puccj**

### CODE REVIEW

File: Statistical Mechanics.pdf
Comment (Grufoony) [inline Statistical Mechanics.pdf:1]: The name should be `Statistical_Mechanics.pdf` (however, it's not that important).

File: src/Statistical_Mechanics/3quantumSM.tex [hunk 1]
```diff
@@ -0,0 +1,1395 @@
...
+    But we also know that $0 \ne \Delta S = \Delta Q / T_c$. So the process of of decaying from $\epsilon > 0$ to $\epsilon =0$ costs some energy, some heat. This heat $\Delta Q \ne 0$ is called latent heat.
+
+    This is a peculiar phase transition, with continuous thermodynamic potentials, but latent heat as (for instance) in evaporation.
+
+    \gray{It is the same as water boiling, which stops at $T= 100^oC$ since particles absorb latent heat.}
+\end{itemize}
+
+\notedate{(XII) MON (ex.3) 12/12/2022}
+\notedate{(XIII) MON (ex.4) 19/12/2022}
+\subsection{Exercise (2.5): Gas of photons}
```
Comment (Grufoony) [inline src/Statistical_Mechanics/3quantumSM.tex]: I guess this should not be empty... in general it's a superb work, well done!
If you don't have this part just tell me and I will merge the PR.

**Review by Grufoony — COMMENTED** [review]
Great work!
Can you look for these two things?
Anyway, if you don't have time to don't warry, just tell me and I'll merge the PR.

**Comment by puccj** [comment]
Done: fixed the name and add the last exercise.

**Review by Grufoony — APPROVED** [review]

**Pull request merged**